### ISSUE TITLE
allow using ciphers provided by other plugins

### ISSUE THREAD

**Issue by deathtrip** [titasraha/multicipher#1]
This plugin is good for people who want maximum security, so imo it would make sense to allow using ciphers provided by other keepass plugins (twofish, serpent etc.). As they are stronger than chacha20 and aes.

**Comment by RyanMDeveau** [comment]
I agree, I think adding the option of additional algorithms like Twofish, Serpent, and GOST R 34.12-2015. It is appreciated that you've provided a functional Yubikey Challenge though, so many thanks for that.

**Comment by titasraha** [comment]
I agree too. Unfortunately I don't get much time to work on it, definitely something I would like to do.

**Comment by titasraha** [comment]
Added support for Twofish algorithm

**Comment by RyanMDeveau** [comment]
This is really fantastic to hear, we appreciate your work on this! :)
